StepDefinitions/SkillTestStepDefinition.cs

```diff
@@ -0,0 +1,84 @@
+﻿using MarsQA.Helpers;
+using MarsQA.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsQA.StepDefinitions
+{
+    [Binding]
+    public class SkillTestStepDefinitions : Driver
+
+    {
+        SkillPage skillPage = new SkillPage();
```

Comment:
same comments apply to this class

Reply:
I have code change to inherit  driver class from the pages and remove from action method